src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -917,6 +917,11 @@ static int do_list_mapped_devices()
 
 static int parse_args(vector<const char*>& args, std::ostream *err_msg, Config *cfg)
 {
+  std::string conf_file_list;
+  std::string cluster;
+  CephInitParameters iparams = ceph_argparse_early_args(
```

Comment:
How we are using this local variable in this function?

Reply:
@amitkumar50 thanks for comment. As @trociny said, I will use it later.